Umbraco Cloud release notes for January (#4870)
umbraco/UmbracoDocs: umbraco-cloud/release-notes/2023-01-releasenotes.md

# Release Notes, January 2023

## Key Takeaways

* **UK Region Hosting** - Umbraco Cloud expands with UK regional hosting.

> * **Display of Multi-factor authentication setting for organisation members** - As an administrator at a cloud organization, you can see the multi-factor authentication setting for all the organization's members.

Reviewer: ```suggestion
* **Display of Multi-factor authentication setting for organization members** - As an administrator at a cloud organization, you can see the multi-factor authentication setting for all the organization's members.
```

> * **Display of Multi-factor authentication setting for organisation members** - As an administrator at a cloud organization, you can see the multi-factor authentication setting for all the organization's members.

Reviewer: ```suggestion
* **Display of Multi-factor authentication setting for organisation members** - As an administrator at a Cloud organization, you can see the multi-factor authentication setting for all the organization's members.
```

> * **Prefixes for naming secrets** - When naming secrets, you can reference the security related settings for Umbraco:CMS:Global:Smtp and Umbraco Forms api-key and field recaptchas.

Reviewer: ```suggestion
* **Prefixes for naming secrets** - When naming secrets, you can reference the security-related settings for Umbraco:CMS:Global:Smtp, Umbraco Forms API-key, and field reCAPTCHAS.
```

## UK Region Hosting

> With an increasing interest in using Umbraco Cloud as the development and hosting platform for Umbraco projects, we’ve also seen an increased demand for additional regional hosting options. Especially when it comes to privacy and data policy requirements, the physical location of the data matters.

Reviewer 2: ```suggestion
With an increasing interest in using Umbraco Cloud as the development and hosting platform for Umbraco projects. We’ve also seen an increased demand for additional regional hosting options. Especially when it comes to privacy and data policy requirements, the physical location of the data matters.
```

> With an increasing interest in using Umbraco Cloud as the development and hosting platform for Umbraco projects, we’ve also seen an increased demand for additional (regional) hosting options. Especially when it comes to privacy and data policy requirements, the physical location of the data matters.

Reviewer: ```suggestion
With an increasing interest in using Umbraco Cloud as the development and hosting platform for Umbraco projects, we’ve also seen an increased demand for additional regional hosting options. Especially when it comes to privacy and data policy requirements, the physical location of the data matters.
```

Regional Hosting Options make it possible to choose where your Umbraco Cloud project, including all environments and data, is served and stored.

> Umbraco Cloud has reached yet a new milestone with the support of UK region. This is fantastic news for everyone working with or out of Britain. 

Reviewer: ```suggestion
Umbraco Cloud has reached a new milestone with the addition of a UK regional hosting option. This is fantastic news for everyone based in and working with clients in the UK. 
```

![Cloud-artboard](images/umbraco-cloud-update-artboard-1-2x.png)

> You are now able to create a new Umbraco Cloud project in "West Europe", "US East", or "UK South" to the delight of the majority of our European and American customers.

Reviewer: ```suggestion
This means can now choose to create new Umbraco Cloud projects in "West Europe", "US East", and "UK South".
```

> ## Display of Multi-factor authentication setting for organisation members

Reviewer: ```suggestion
## Display of Multi-factor Authentication Setting for Organization Members
```

> Since October last year, our users have had the opportunity to enable multi-factor authentication (MFA) and thereby authenticate with a password and either phone, email or an authenticator app.

Reviewer: ```suggestion
Since October last year, our users have had the opportunity to enable multi-factor authentication (MFA) and thereby authenticate with a password and either phone, email, or an authenticator app.
```

> Since October last year, our users have had the opportunity to enable multi-factor authentication (MFA) and thereby authenticate with a password and either phone, email or an authenticator app.

Reviewer 2: ```suggestion
Since October last year, users has the opportunity to enable Multi-Factor Authentication (MFA) and authenticate with either phone, email, or an authenticator app.
```

> Now we deliver the second part of the planned three steps of the MFA feature, by ensuring that an administrator in a cloud organization can see the multi-factor authentication settings for all organization members.

Reviewer: ```suggestion
We have now released the second part of the three steps planned for the MFA feature, by ensuring that an administrator in a Cloud organization can see the multi-factor authentication settings for all organization members.
```

> Now we deliver the second part of the planned three steps of the MFA feature, by ensuring that an administrator in a cloud organization can see the multi-factor authentication settings for all organization members.

Reviewer 2: ```suggestion
Now we deliver the second part of the planned three steps of the MFA feature. We ensure that administrators in a cloud organization can see the multi-factor authentication settings for all organization members.
```

![OrgMfaDisplay](images/OrgMfaDisplay2.png)

> The third and final step will be the enforcement of MFA per user level based on an organization setting. This means that an administrator can set the desired MFA type (email, phone or authenticator app) for each individual user, which will then be imposed on the user at the next login to Umbraco Cloud. The enforcement of multi-factorr authentication for organisation members is expected to be delivered in the first quarter of 2023.

Reviewer: ```suggestion
The third and final step will be the enforcement of MFA on a per-user level based on organization settings. This means that an administrator can set the desired MFA type (email, phone, or authenticator app) for each individual user, which will then be imposed on the user at the next login to Umbraco Cloud. The enforcement of multi-factor authentication for organization members is expected to be delivered in the first quarter of 2023.
```

> The third and final step will be the enforcement of MFA per user level based on an organization setting. This means that an administrator can set the desired MFA type (email, phone or authenticator app) for each individual user, which will then be imposed on the user at the next login to Umbraco Cloud. The enforcement of multi-factorr authentication for organisation members is expected to be delivered in the first quarter of 2023.

Reviewer 2: ```suggestion
The third and final step will be the enforcement of MFA per user level based on an organization setting. This means that an administrator can set the desired MFA type (email, phone, or authenticator app) for each individual user. This will then be imposed on the user at the next login to Umbraco Cloud. The enforcement of multi-factor authentication for organization members is expected to be delivered in the first quarter of 2023.
```

> ## [Prefixes for naming secrets](https://docs.umbraco.com/umbraco-cloud/set-up/project-settings/secrets-management)

Reviewer: ```suggestion
## [Prefixes for Naming Secrets](https://docs.umbraco.com/umbraco-cloud/set-up/project-settings/secrets-management)
```

> In December the Secrets management feature was released in Umbraco Cloud, which enables all Cloud projects on the Standard and Pro to securely store secrets in Azure key vaults and reference them and access simply and conveniently during the runtime of the application.

Reviewer: ```suggestion
In December the Secrets Management feature was released in Umbraco Cloud, enabling all Cloud projects on the Standard and Pro plans to securely store secrets in Azure key vaults and manage them in the Cloud portal. These can then be referenced and accessed during the runtime of the application on a per-environment basis.
```

> In December the Secrets management feature was released in Umbraco Cloud, which enables all Cloud projects on the Standard and Pro to securely store secrets in Azure key vaults and reference them and access simply and conveniently during the runtime of the application.

Reviewer 2: ```suggestion
In December the Secrets management feature was released in Umbraco Cloud. This enables all Cloud projects on the Standard and Pro to securely store secrets in Azure key vaults. It is possible to reference them and access them simply and conveniently during the runtime of the application.
```

> In December the Secrets Management feature was released in Umbraco Cloud, enabling all Cloud projects on the Standard and Pro plans to securely store secrets in Azure key vaults and manage them in the Cloud portal. These can then be referenced and accessed during the runtime of the application on a per-environment basis.

Reviewer 2: ```suggestion
In December the Secrets Management feature was released in Umbraco Cloud. This enables Cloud projects on the Standard and Pro plans to securely store secrets in Azure key vaults and manage them in the Portal. These can then be referenced and accessed during the runtime of the application on a per-environment basis.
```

> The naming of secrets is subject to some restrictions to ensure that reserved words and application settings cannot be accidentally overwritten.

Reviewer: ```suggestion
Naming secrets is subject to some restrictions to ensure that reserved words and application settings cannot be accidentally overwritten.
```

> Certain application settings should, however, be overwritten precisely under the auspices of security. Whereby it is now possible to overwrite the following three namespaces

Reviewer: Not sure I understand this 🤔

- Umbraco:CMS:Global:Smtp
- Umbraco:Forms:Security:FormsApiKey
- Umbraco:Forms:FieldTypes:Recaptcha  

> There namespaces are relevant to consider when adding a SMTP to the Umbraco CMS, saving the Umbraco Forms API key, or using the ReChaptcha for Umbraco Forms.

Reviewer 2: ```suggestion
These namespaces are relevant when adding an SMTP to the Umbraco CMS, saving the Umbraco Forms API key, or using ReChaptcha for Umbraco Forms.
```

> These namespaces are relevant when adding an SMTP to the Umbraco CMS, saving the Umbraco Forms API key, or using ReChaptcha for Umbraco Forms.

Reviewer 2: ```suggestion
These namespaces are relevant when adding a Simple Mail Transfer Protocol (SMTP) to the Umbraco CMS, saving the Umbraco Forms API key, or using ReChaptcha for Umbraco Forms.
```
